universe/readers/event_readers.py

```diff
@@ -0,0 +1,484 @@
+import heapq
+import json
+import logging
+
+from universe import error
+from universe import spaces
+from universe import utils
+from universe.vncdriver import fbs_reader, vnc_proxy_server, vnc_client
+from universe import spaces as vnc_spaces
+
+logger = logging.getLogger(__name__)
+
+class InvalidEventLogfileError(error.Error):
```

Comment:
Add another blank line. There are a lot of lint error, consider running flake8 before pushing the code.